afids_utils/tests/strategies.py

```diff
@@ -120,3 +120,24 @@ def affine_xfm(
         affine = matrices[xfm] @ affine
 
     return affine
+
+
+@st.composite
+def nii_img(
+    draw: st.DrawFn,
+    float: bool = True,
+    shape: tuple[int, int, int] = (16, 16, 16),
+    min_value: int = 0,
```

Comment:
The type signature here prevents non-integral min/max values for float arrays. I'd change the type hint to `float` and cast the values to `int` if `float` is False.

Reply:
I'll need to double check again, but it shouldn't constrain the of the values within the array to `int` (which ends up using `st.floats` to draw the values between the min and max).

Reply:
Ah I'm just saying if I want an array of floats between 0.1 and 0.5 there's no way for me to express that right now

Reply:
Ahh I see what you mean.

Reply:
Really I think this type signature should have an overload where `min_value` and `max_value` are floats if float=True but ints if `float=False`, now that I think about it